src/webui/webapplication.h

```diff
@@ -38,6 +38,7 @@ class WebApplication: public AbstractWebApplication
 
 public:
     explicit WebApplication(QObject* parent = 0);
+    using AbstractWebApplication::processRequest;
```

Comment:
> How can some method to hide a method with a different signature?

Compiler does not consider base class members when resolving overloads.

> In this case it's better to rename private one. Maybe processRequest_impl?

I don't like the idea of abstract methods named xxx_impl. Propose `doProcessRequest()`.